_commands/os_x/ditto.md

```diff
@@ -0,0 +1,72 @@
+---
+---
+
+Ditto
+-------
+
+Ditto is an OSX command that copies the entire contents of a working directory to another, new location.
+
+~~~ bash
+ditto /old/directory/ /new/directory/
+~~~
+
+<!--more-->
+
+### Details
+Ditto perserves file resources forks and metadata, in addition to ownership attributes. This ensures a more exact copy than cp. Ditto can also be used to copy a file or direct to a source that doesn't exist - in that case, ditto will automatically create the source for the user.
+
+
+### Useful Options / Examples
+
+#### -V 
+~~~ bash
+ditto -V old new
+>>>copying old
+copying file ./bla.txt ...
+10 bytes for ./bla.txt
+~~~
+
+#### Break it down
+-V stands for verbose, and is a flag that prints 2 lines for every file being copied: one line displaying the name of the file, and one displaying the file size.
+
+#### --norsrc
+~~~ bash
+ditto --nosrc /old /new
+~~~
+
+#### Break it down
+The --norsrc flag allows you to copy files and directories without copying their accompanying metadata and resource forks.
+
+#### --password
+~~~ bash
+ditto --password /old /new
+~~~
+
```

Comment:
all of the lines with flags are only displaying one hyphen when there are two in the markdown. Personally, I'd put all of these headers in backticks (```) because they're command line options and `backticks_indicate_inline_code_stuff`. It's also consistent with the rest of the documentation submissions.